flexmeasures/cli/data_add.py

```diff
@@ -1047,8 +1096,7 @@ def add_toy_account(kind: str, name: str):
         # make an account (if not exist)
         account = Account.query.filter(Account.name == name).one_or_none()
         if account:
-            click.echo(f"Account {name} already exists.")
-            return
+            click.echo(f"Account already exists: {account}")
```

Comment:
I believe if the toy account already exists, I wouldn't start creating things in it. Let's advise to delete it first.

I can't think of a use case where you want to "freshen up" the account structure, unless you're developing this CLI, then you can comment this out.